core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -63,13 +73,26 @@
 
 @SuppressWarnings("deprecation") // Added for Scala 2.12 compatibility for usages of JavaConverters
 @ExtendWith(value = ClusterTestExtensions.class)
+@ClusterTestDefaults
 @Tag("integration")
 public class ConfigCommandIntegrationTest {
-    AdminZkClient adminZkClient;
-    List<String> alterOpts;
 
+    private List<String> alterOpts;
+    private final String defaultBrokerId = "0";
     private final ClusterInstance cluster;
 
+    private static Runnable run(Stream<String> command) {
+        return () -> {
+            try {
+                ConfigCommand.main(command.toArray(String[]::new));
+            } catch (RuntimeException e) {
```

Comment:
This is redundant if you do nothing in catching.

Reply:
If removed this catch block, the `testExitWithNonZeroStatusOnUpdatingUnallowedConfig` can't test the message, because the RuntimeException doesn't beed catched